---
layout: post
category: publication
published: true
title: "OligoMiner provides a rapid, flexible environment for the design of genome-scale oligonucleotide in situ hybridization probes"
date: 2018-03-06 12:00:00
pmid: 29463736
authors: "Beliveau BJ, Kishi JY, Nir G, Sasaki HM, Saka SK, Nguyen SC, Wu CT, Yin P"
firstauthor: "Beliveau BJ"
journalname: "Proc. Natl. Acad. Sci. U. S. A."
journalvolume: 115
journalissue: 10
journalpages: E2183â€“E2192
---

Oligonucleotide (oligo)-based FISH has emerged as an important tool for the study of chromosome organization and gene expression and has been empowered by the commercial availability of highly complex pools of oligos. However, a dedicated bioinformatic design utility has yet to be created specifically for the purpose of identifying optimal oligo FISH probe sequences on the genome-wide scale. Here, we introduce OligoMiner, a rapid and robust computational pipeline for the genome-scale design of oligo FISH probes that affords the scientist exact control over the parameters of each probe. Our streamlined method uses standard bioinformatic file formats, allowing users to seamlessly integrate new and existing utilities into the pipeline as desired, and introduces a method for evaluating the specificity of each probe molecule that connects simulated hybridization energetics to rapidly generated sequence alignments using supervised machine learning. We demonstrate the scalability of our approach by performing genome-scale probe discovery in numerous model organism genomes and showcase the performance of the resulting probes with diffraction-limited and single-molecule superresolution imaging of chromosomal and RNA targets. We anticipate that this pipeline will make the FISH probe design process much more accessible and will more broadly facilitate the design of pools of hybridization probes for a variety of applications.

URL: [http://dx.doi.org/10.1073/pnas.1714530115](http://dx.doi.org/10.1073/pnas.1714530115)
